Applications/Spire/Source/KeyBindings/KeyBindingsWindow.cpp

```diff
@@ -9,11 +9,33 @@
 using namespace boost::signals2;
 using namespace Spire;
 
+namespace {
+  auto make_button(const QString& text, QWidget* parent) {
```

Comment:
I may be wrong on this, but in C++ `make_` prefix is most often used for "external constructors", e.g. functions forwarding arguments to the actual constructor of a class for a certain benefit (most often, template deduction and exception safety).
In this case, you are adding some non-trivial, application-specific logic to button creation, so I would give it a different name, e.g. `create_button` (or `add_button`, as you are technically adding a button to the parent widget - it is recommended to swap the arguments in that case though), to be safe.

Reply:
I can change it to create_button, I usually consider add_x to be adding an item to a layout or view.

The last parameter being the parent is a convention that I borrowed from Qt.

Reply:
OK.